addons/l10n_in/models/template_in.py

```diff
@@ -47,3 +48,61 @@ def _get_in_account_cash_rounding(self):
                 'loss_account_id': 'p213201',
             }
         }
+
+    @template('in', 'account.fiscal.position')
+    def _get_in_account_fiscal_position(self):
+        company = self.env.company
+        state_ids = [Command.set(company.state_id.ids)] if company.state_id else False
+        intra_state_name = company.state_id and _('Within %s', company.state_id.name) or _('Intra State')
+        state_specific = {
+            'fiscal_position_in_intra_state': {
+                'name': intra_state_name,
+                'sequence': 1,
+                'auto_apply': True,
+                'state_ids': state_ids,
+                'country_id': self.env.ref('base.in').id,
+            },
+            'fiscal_position_in_inter_state': {
+                'name': _('Inter State'),
+                'sequence': 2,
+                'auto_apply': True,
+                'tax_ids': self._get_l10n_in_fiscal_tax_vals(),
+                'country_id': self.env.ref('base.in').id
+            },
+        }
+        if company.parent_id:
+            return state_specific
+        return {
+            **state_specific,
+            'fiscal_position_in_outside_in': {
+                'name': _('Outside India'),
+                'sequence': 3,
+                'auto_apply': True,
+                'tax_ids': self._get_l10n_in_fiscal_tax_vals(),
+            },
+            'fiscal_position_in_sez': {
+                'name': _('Special Economic Zone (SEZ)'),
+                'sequence': 4,
+                'tax_ids': self._get_l10n_in_fiscal_tax_vals(),
+            },
+            'fiscal_position_in_lut_sez': {
+                'name': _('LUT - Special Economic Zone (SEZ)'),
+                'sequence': 5,
+                'tax_ids': self._get_l10n_in_fiscal_tax_vals(use_zero_rated_igst=True),
+            },
+            'fiscal_position_in_lut_outside_in': {
+                'name': _("LUT Outside India"),
+                'sequence': 6,
+                'tax_ids': self._get_l10n_in_fiscal_tax_vals(use_zero_rated_igst=True),
+            },
```

Comment:
What's the difference between those 2 fiscal positions?

Reply:
@ajjo-odoo @jva-odoo @hamo-odoo

Reply:
About **LUT Outside India** :
 Letter of Undertaking holds significant relevance within the context of the Goods and Services Tax (GST) framework. This document serves as a powerful tool for exporters, allowing them to engage in the export of goods or services without the obligation of immediate tax payment

About **LUT - SEZ** :
SEZ units can make inward supplies in two ways as follows:

- Without payment of IGST (With LUT) 
 - With payment of IGST (Without LUT) and claiming Input Tax Credit

The suppliers of SEZ unit/developer are required to issue LUT so they don’t need to collect GST.  

However, if they don’t issue LUT, they have to collect IGST and then SEZ unit/developer can avail refund of such Input Tax Credit under GST

cc- @ajjo-odoo

Reply:
Ok, but why do we need 2 separate fiscal positions?
Does it make sense? Will they ever be different?

Reply:
> Ok, but why do we need 2 separate fiscal positions?

For Functional purposes both will be treated different for reporting

> Does it make sense? Will they ever be different?

Technically not but yes for functional reporting of both fiscal will be different

Reply:
It is preferable not to use the use of fiscal position for that, because you can always put the taxes manually without the fiscal position.  Should not we have separate taxes for it if they are reported differently?  @ajjo-odoo

Reply:
1) Within Gujarat ( company state )    (Intra State if not configured; rename when company state changes) (No tax mapping)
==> If supply made between same state ( company state = partner state ) Exception if SEZ.

2) Inter-state (GST -> IGST)
==> Company State is diffrent than Partner State ( In case of India ) 

3) Special Economic Zone (SEZ) (GST -> IGST)
==> State could be same or different - Government declare specific area as a SEZ where in supply
as fiscal position does not support parameter of GST Treatment type to auto apply so, it set for manuall selection.

4) Outside India without LUT (GST -> IGST)
==> Country is not equal to India - here supplier will pay a Tax to Government on supply of goods and will get credit back

5) Outside India with LUT (GST -> 0% IGST)

==> Country is not equal to India - here supplier will not pay a tax as he will hold a certificate of LUT and tax rate would be Zero percent.

As LUT is based on period and certain conditions so posisbility of supply with LUT & Without for the same company.
So it would be good to keep both.

Reply:
@ajjo-odoo What about just one fiscal position "LUT"?

Reply:
> @ajjo-odoo What about just one fiscal position "LUT"?

yeah thanks for the idea, it's fine to have as per our last call.